### ISSUE TITLE
read manifest from mirror(https://tiup-mirrors.pingcap.com/) failed: not enough signatures (0) for threshold 1 in timestamp.json

### ISSUE THREAD

**Issue by Tongzhenguo** [pingcap/tiup#524]
## Bug Report

tiup version v1.0.1 tiup
Go Version: go1.13
Git Branch: v1.0
GitHash: 75d112b

vi scale-out.yaml
```
tikv_servers:
  - host: 10.16.16.93
    ssh_port: 34185
    port: 20160
    status_port: 20180
    deploy_dir: /data/deploy/install/deploy/tikv-20160
    data_dir: /data/deploy/install/data/tikv-20160
    log_dir: /data/deploy/install/log/tikv-20160
pd_servers:
  - host: 10.16.16.93
    ssh_port: 34185
    name: pd-4
    client_port: 2379
    peer_port: 2380
    deploy_dir: /data/deploy/install/deploy/pd-2379
    data_dir: /data/deploy/install/data/pd-2379
    log_dir: /data/deploy/install/log/pd-2379  
```

```
tiup cluster scale-out tidb-test2 /www/scale-out.yaml
```

error log as follow:

```
020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=pd, version=v4.0.0, os=linux, arch=amd64\nDownload: component=tikv, version=v4.0.0, os=linux, arch=amd64\nDownload: component=node_exporter, version=v0.17.0, os=linux, arch=amd64\nDownload: component=blackbox_exporter, version=v0.12.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=blackbox_exporter, version=v0.12.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=tikv, version=v4.0.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=blackbox_exporter, version=v0.12.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=pd, version=v4.0.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=node_exporter, version=v0.17.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=node_exporter, version=v0.17.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=tikv, version=v4.0.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.492+0800 DEBUG TaskBegin {“task”: “Download: component=pd, version=v4.0.0, os=linux, arch=amd64”}
2020-06-22T16:03:44.587+0800 DEBUG TaskFinish {“task”: “Download: component=tikv, version=v4.0.0, os=linux, arch=amd64”, “error”: “read manifest from mirror(https://tiup-mirrors.pingcap.com/) failed: not enough signatures (0) for threshold 1 in timestamp.json”, “errorVerbose”: “not enough signatures (0) for threshold 1 in timestamp.json\ngithub.com/pingcap/tiup/pkg/repository/v1manifest.(*KeyStore).verifySignature\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1manifest/key_store.go:143\ngithub.com/pingcap/tiup/pkg/repository/v1manifest.ReadManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1manifest/manifest.go:397\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchManifest.func1\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:466\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchBase\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:494\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:465\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).checkTimestamp\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:422\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).updateLocalSnapshot\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:217\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).ensureManifests\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:163\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).FetchComponentManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:568\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).ComponentVersion\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:578\ngithub.com/pingcap/tiup/pkg/cluster/clusterutil.(*repositoryT).DownloadComponent\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/clusterutil/cluster.go:55\ngithub.com/pingcap/tiup/pkg/cluster/operation.Download\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/operation/download.go:59\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Downloader).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/download.go:43\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Serial).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/task.go:183\ngithub.com/pingcap/tiup/pkg/cluster/task.(*StepDisplay).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/step.go:85\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Parallel).Execute.func1\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/task.go:236\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357\nread manifest from mirror(https://tiup-mirrors.pingcap.com/) failed”}
2020-06-22T16:03:44.587+0800 DEBUG TaskFinish {“task”: “Download: component=blackbox_exporter, version=v0.12.0, os=linux, arch=amd64”, “error”: “read manifest from mirror(https://tiup-mirrors.pingcap.com/) failed: not enough signatures (0) for threshold 1 in timestamp.json”, “errorVerbose”: “not enough signatures (0) for threshold 1 in timestamp.json\ngithub.com/pingcap/tiup/pkg/repository/v1manifest.(*KeyStore).verifySignature\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1manifest/key_store.go:143\ngithub.com/pingcap/tiup/pkg/repository/v1manifest.ReadManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1manifest/manifest.go:397\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchManifest.func1\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:466\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchBase\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:494\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).fetchManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:465\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).checkTimestamp\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:422\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).updateLocalSnapshot\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:217\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).ensureManifests\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:163\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).FetchComponentManifest\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:568\ngithub.com/pingcap/tiup/pkg/repository.(*V1Repository).ComponentVersion\n\t/home/jenkins/agent/workspace/tiup-release/pkg/repository/v1_repository.go:578\ngithub.com/pingcap/tiup/pkg/cluster/clusterutil.(*repositoryT).DownloadComponent\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/clusterutil/cluster.go:55\ngithub.com/pingcap/tiup/pkg/cluster/operation.Download\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/operation/download.go:59\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Downloader).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/download.go:43\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Serial).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/task.go:183\ngithub.com/pingcap/tiup/pkg/cluster/task.(*StepDisplay).Execute\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/step.go:85\ngithub.com/pingcap/tiup/pkg/cluster/task.(*Parallel).Execute.func1\n\t/home/jenkins/agent/workspace/tiup-release/pkg/cluster/task/task.go:236\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357\nread manifest from mirror(https://tiup-mirrors.pingcap.com/) failed”}
```

**Comment by lucklove** [comment]
This will be fixed in #503

**Comment by lucklove** [comment]
This has been fixed.
Next time you can use the command `tiup mirror set https://tiup-mirrors.pingcap.com` to change mirror and this action will avoid this issue.
